extensions-core/parquet-extensions/src/test/java/org/apache/druid/data/input/parquet/DecimalParquetInputTest.java

```diff
@@ -148,13 +148,12 @@ public void testReadParquetDecimali32() throws IOException, InterruptedException
   public void testReadParquetDecimali64() throws IOException, InterruptedException
   {
     // parquet-avro does not correctly convert decimal types
-    if (parserType.equals(ParquetExtensionsModule.PARQUET_AVRO_INPUT_PARSER_TYPE)) {
-      return;
-    }
+    assumeFalse(parserType.equals(ParquetExtensionsModule.PARQUET_AVRO_INPUT_PARSER_TYPE));
+
     HadoopDruidIndexerConfig config = transformHadoopDruidIndexerConfig(
-        "example/decimals/dec_in_i64.json",
-        parserType,
-        true
+            "example/decimals/dec_in_i64.json",
```

Comment:
These changes are not needed as this code is already formatted correctly according to the formatting style used by this project. Same comment for the other formatting changes.